### ISSUE TITLE
Warning message on admin

### ISSUE THREAD

**Issue by DanielSousa** [checkout/checkout-magento2-plugin#237]
Magento version: 2.3.0
checkoutcom version: 2.2.2

Steps to reproduce:

1. Install the extension in developer mode
```
composer require checkoutcom/magento2:^2.2.2
bin/magento setup:upgrade
rm -rf var/cache var/generation/ var/di
php bin/magento cache:clean
```

2. Login in to admin

An warning message is shown 
```
Warning: count(): Parameter must be an array or an object that implements Countable in /var/www/html/vendor/checkoutcom/magento2/Model/Service/VersionHandlerService.php on line 143
```

**Comment by luke-fenn-cko** [comment]
Hello,

Thank you for reporting this.

This issue is specific to version 2.2.2 of the plugin and has already been noted, with a fix coming in the next release. 

If I might provide some reassurance however, it will only affect the message queue in Magento admin and should not cause any issues with taking payment.

Many thanks,
Luke

**Comment by david-fiaty-cko** [comment]
@vikasi95dev, @DanielSousa, this issue should now be fixed in the 2.2.3 RC version:
https://github.com/checkout/checkout-magento2-plugin/tree/2.2.3-candidate

This is in now in active QA. You will be informed as soon as the official release is available but please feel free to test the RC.

**Comment by david-fiaty-cko** [comment]
@vikasi95dev, @DanielSousa , there is a new release with a fix for this issue:
https://github.com/checkout/checkout-magento2-plugin/releases/tag/2.2.3

**Comment by david-fiaty-cko** [comment]
Closing this issue as it's now fixed in the 2.2.3 release.
